me.ahola.aphototoollibre.yml

```diff
@@ -0,0 +1,27 @@
+app-id: me.ahola.aphototoollibre
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: aphototoollibre
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
it's either `x11` alone, or `fallback-x11` with wayland

Reply:
Copied from here https://docs.flatpak.org/en/latest/qt.html

Should line 8 be
 - --socket=fallback-x11
 
?

Reply:
this doc is wrong and doesn't ask the right questions.